webapp/src/components/common/Form/index.tsx

```diff
@@ -116,6 +116,7 @@ function Form<TFieldValues extends FieldValues, TContext>(
   const { t } = useTranslation();
   const autoSubmitConfig = toAutoSubmitConfig(autoSubmit);
 
+  const [isInProgress, setIsInProgress] = useState(false);
```

Comment:
consider this instead: `  const [isSubmitting, setIsSubmitting] = useState(false);`

Reply:
`isSubmitting` already exist...

Reply:
In the same component? Ok mybad didnt see that

Reply:
maybe you can just add the 'submit' to indicate it refers to a submitting state like `isSubmitInProgress` (it's a detail tho feel free to skip)